Replaced the URLs for Firefox and CURL users

mjivraja opened the pull request:
The URLs that are mentioned at present are incorrect and return
the 404 error because of http:// instead of https://. Updated 
both URLs with the new ones along with https:// as shared by the reporter.
Also, added the type of URL in the address bar of Firefox users.

Fix the url protocol and end points in Section 3.4 Browsing Repository
Content Using an Organization Debug Certificate

https://bugzilla.redhat.com/show_bug.cgi?id=2097382


Cherry-pick into:

* [X] Foreman 3.3
* [X] Foreman 3.2
* [X] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

<!---
Thank you for contributing to Foreman documentation. Make sure to read README
for the documentation standards. Set cherry-pick github label to mark this
contribution for cherry picking and check which version do you need with [x].
-->

evgeni commented on guides/common/modules/proc_browsing-repository-content-using-an-organization-debug-certificate.adoc:
```diff
@@ -36,11 +36,11 @@ $ openssl pkcs12 -keypbe PBE-SHA1-3DES -certpbe PBE-SHA1-3DES -export -in cert.p
 [options="nowrap" subs="+quotes,attributes"]
 ----
-http://_{foreman-example-com}_/pulp/repos/_organization_label_
+https://_{foreman-example-com}_/pulp/content/organization_label/Library/content/dist/rhel/server/7/7Server/x86_64/os/_
```
Should this be
```suggestion
https://_{foreman-example-com}_/pulp/content/_organization_label_/Library_/content/dist/rhel/server/7/7Server/x86_64/os/
```

so that `_organization_label_` is still italics? also, using 7Server explicitly seems like an odd choice to me?

ehelms replied:
> Yeah, the "problem" is that in pulp2, you could go to `/pulp/repos/<org>` and see what you have -- but on pulp3 you can't do that, and get only a 404. What you can do is to go to `/pulp/content/` and see _everything_, which I think is better than pointing to an explicit repo (even if existing).
> 
> Honestly, this feels like a missing feature (or config error?) in pulp3 to me, but we're not going to fix that trivially today, so probably should document what's working today.

I agree with this as that is part of the power of browsing with the debug certificate. Including one directed example works in addition to this; noting you can visit the root `pulp/content` to see all available items to look at is worth a note.

sayan3296 replied:
+1 I have commented in the BZ now and I believe, It's good to cover the `/pulp/content/` part there in Point 5 i.e.

~~~
In the address bar, enter the following URL to browse the accessible paths for all the repositories and further check their contents:

https://{foreman-example-com}/pulp/content/
~~~

@mjivraja FYI

mjivraja replied:
So, by going through the comments, I assume there is no modification required for URLs. The only thing is keeping `_organization_label_` in italics.

@evgeni @ehelms @sayan3296 Please confirm.

sayan3296 replied:
@mjivraja I hope after our small discussion on g-chat, You are clear about the newest suggestion made above.

mjivraja replied:
Yes. Thank you for clarifying 👍

mjivraja replied:
The Builds were successful locally for all. This is a new error for me. :worried:

mjivraja commented:
> @mjivraja if you rebase on current `master`, the `preview` build error will go away

I pulled `master` and rebased my feature branch. Also, squashed commit.

evgeni commented on guides/common/modules/proc_browsing-repository-content-using-an-organization-debug-certificate.adoc:
```diff
@@ -51,6 +51,6 @@ To use the organization debug certificate with CURL, enter the following command
 ----
 $ curl -k --cert cert.pem --key key.pem \
-http://_{foreman-example-com}_/pulp/repos/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/sat-tools/{ProductVersion}/os/
+https://_{foreman-example-com}_/pulp/content/_organization_label_/Library/content/dist/rhel/server/7/7Server/x86_64/os/
```
If we change `_organization_label_` to `_My_Organization_Label_` here, we should also change it in line 33 for consistency:
```
$ openssl pkcs12 -keypbe PBE-SHA1-3DES -certpbe PBE-SHA1-3DES -export -in cert.pem -inkey key.pem -out _organization_label_.pfx -name _My_Organization_
```

(other files are also using the `_My_Organization_Label_` notation, so it seems a good idea to make that change here)

evgeni commented:
> @mjivraja if you rebase on current `master`, the `preview` build error will go away

It did not, but it's not your fault. Ignore this for now please.

maximiliankolb reviewed (approved):
LGTM!

maximiliankolb commented:
Merged to master and cherry-picked: ✅ 
   023c4f6f7..fefe8380d  3.3 -> 3.3
   d273f08dc..e4ba88ee9  3.2 -> 3.2
   56362747f..47700e1a1  3.1 -> 3.1
